### DISCUSSION TITLE
How to globally change the border-radius?

### DISCUSSION THREAD

**Discussion by marrobHD** [saadeghi/daisyui#2491] [Q&A] [answered]
Hi,

I'd like to know if there's a way to globally adjust the border-radius.

Thanks :>

**Comment by saadeghi** [comment, marked as answer]
There are CSS variables: https://daisyui.com/docs/themes/#-5
```js
"--border-btn": "1px", // border width of buttons
"--tab-border": "1px", // border width of tabs
"--tab-radius": "0.5rem", // border radius of tabs
```
You can set them in a daisyUI theme  
or you can use them directly in CSS  
or you can use the in utility class like `[--border-btn:2rem]`

**Reply by lefpap** [reply]
Is there a way to view to view the existing values of a theme?
